optaplanner-docs/src/main/asciidoc/DroolsScoreCalculation/DroolsScoreCalculation-chapter.adoc

```diff
@@ -301,3 +301,98 @@ end
 Notice how `addSoftConstraintMatch()` specifies that it's a soft constraint,
 and needs a negative number to penalize each match. Otherwise it would reward such matches.
 The parameter `($capacity - $studentSize)` always results in a negative number because `studentSize > $capacity`.
+
+
+[[testingDroolsConstraints]]
+== Testing Drools-based constraints
+
+Drools-based constraints come with a unit testing harness.
+To use it, first add a test scoped dependency to the `optaplanner-test` jar to take advantage of the JUnit integration
+and use the `ScoreVerifier` classes to test score rules in DRL (or a constraint match aware incremental score calculator).
+For example, suppose we want to test these score rules:
```

Comment:
"we"

Reply:
ah, it's in my old writing. In any case, if you could replace it, that would be nice :)